modules/fscloud/variables.tf

```diff
@@ -0,0 +1,147 @@
+##############################################################################
+# Input Variables
+##############################################################################
+
+variable "resource_group_id" {
+  type        = string
+  description = "The resource group ID where the RabbitMQ instance will be created."
+}
+
+variable "instance_name" {
+  type        = string
+  description = "The name of the RabbitMQ instance"
+}
+
+variable "plan_validation" {
+  type        = bool
+  description = "Enable or disable validating the database parameters for rabbitmq during the plan phase"
+  default     = true
+}
+
+variable "region" {
+  type        = string
+  description = "The region where you want to deploy your instance."
+  default     = "us-south"
+}
+
+variable "rabbitmq_version" {
+  description = "The version of RabbitMQ to deploy. If no value passed, the current ICD preferred version is used."
+  type        = string
+  default     = null
+}
+
+variable "endpoints" {
+  description = "Endpoints available to the database instance (public, private, public-and-private)"
+  type        = string
+  default     = "private"
+}
+
+variable "tags" {
+  type        = list(any)
+  description = "Optional list of tags to be added to the RabbitMQ instance."
+  default     = []
+}
+
+variable "kms_key_crn" {
+  type        = string
+  description = "The root key CRN of a Key Management Services like Key Protect or Hyper Protect Crypto Service (HPCS) that you want to use for disk encryption. Only used if var.kms_encryption_enabled is set to true."
+  default     = null
+}
+
+variable "existing_kms_instance_guid" {
+  description = "The GUID of the Hyper Protect Crypto Services instance."
+  type        = string
+}
+
+variable "admin_pass" {
+  type        = string
+  description = "The password for the database administrator. If the admin password is null then the admin user ID cannot be accessed. More users can be specified in a user block. The admin password must be in the range of 10-32 characters."
```

Comment:
Can you update the last sentence to:

The admin password must be in the range of 10-32 characters and cannot start with a special character.